GymBro/Views/RoutineViews/RoutineSelectionVIew.swift

```diff
@@ -8,9 +8,86 @@
 
 import SwiftUI
 
+/// Summary
+///
+/// This is a page show up after clicking "new routine" button in NewRoutineVIew. Customer will able to select their desire routnie and  add a new blank reoutn in this page. By clicking any one of the routine button, it will link to the correspond RoutineExerciseView.
+///
+///
 struct RoutineSelectionVIew: View {
+    
+    
+    // MARK: PROPERTY
+    
+    // An example list of routines
+    @State var routines: [String] = [
+        "Leg Day",
+        "Chest Day",
+        "Back Day",
+    ]
+    
+    // A variable that determine whether this page should show a pop-up widow for adding a new routine or not
+    @State var addRoutine: Bool = false
+    
+    
+    // MARK: BODY
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
```

Comment:
The MainView has provided NavView for the Routine module. 
I think we don't need to add one NavView again. 

If removing the NavView doesn't bother the functionality, please remove it.